### ISSUE TITLE
Some problems with php7

### ISSUE THREAD

**Issue by vufa** [mjnaderi/Sharif-Judge#52]
After install and click the login, show some error information:

```
Fatal error: Uncaught Error: Call to a member function row() on boolean in /var/www/html/application/models/Settings_model.php:28 Stack trace: #0 /var/www/html/application/controllers/Login.php(49): Settings_model->get_setting('enable_registra...') #1 /var/www/html/system/core/CodeIgniter.php(356): Login->index() #2 /var/www/html/index.php(285): require_once('/var/www/html/s...') #3 {main} thrown in /var/www/html/application/models/Settings_model.php on line 28
```

```
A PHP Error was encountered

Severity: Error

Message: Uncaught Error: Call to a member function row() on boolean in /var/www/html/application/models/Settings_model.php:28 Stack trace: #0 /var/www/html/application/controllers/Login.php(49): Settings_model->get_setting('enable_registra...') #1 /var/www/html/system/core/CodeIgniter.php(356): Login->index() #2 /var/www/html/index.php(285): require_once('/var/www/html/s...') #3 {main} thrown

Filename: models/Settings_model.php

Line Number: 28

Backtrace:
```

and my php vision is:

```
PHP 7.0.8-0ubuntu0.16.04.2 (cli) ( NTS )
Copyright (c) 1997-2016 The PHP Group
Zend Engine v3.0.0, Copyright (c) 1998-2016 Zend Technologies
    with Zend OPcache v7.0.8-0ubuntu0.16.04.2, Copyright (c) 1999-2016, by Zend Technologies
```

**Comment by erlichsefi** [comment]
@countstarlight  did you solve it?

**Comment by vufa** [comment]
@erlichsefi The problem has been solved before, you need update to CodeIgniter 3.1 and Twig 1.x, and I added some features based on this project. But I have already abandoned the maintenance of this project. Now I'm try to re-development with react.js+golang.
